<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMenu</class>
 <widget class="QMainWindow" name="MyMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="openFileAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <addaction name="showToolAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">background:rgb(170, 255, 255)</string>
       </property>
       <property name="text">
        <string>这是工具箱</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dockLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="closeTool_Btn">
       <property name="text">
        <string>关闭工具箱</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="openFileAction">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="showToolAction">
   <property name="text">
    <string>显示</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
